launchable/commands/record/case_event.py

```diff
@@ -17,7 +17,7 @@ class CaseEvent:
     TestPathBuilder = Callable[[TestCase, TestSuite, str], TestPath]
 
     @staticmethod
-    def default_path_builder(base_path) -> TestPathBuilder:
+    def default_path_builder(file_path_normalizer: FilePathNormalizer) -> TestPathBuilder:
```

Comment:
It's a good design 👍 At first I couldn't understand why you designed FilePathNormalizer as a class, not a function. But later, I was convinced that the class holds basepaths and sets the instance to default path builder.